docs/installation/mac-os.md

```diff
@@ -42,6 +42,20 @@ For additional configuration, [click here](/additional-postgres-setup)
 #### Possible Error Messages
 
 **Error:** `rbenv: version '<version number>' is not installed (set by /Path/To/Local/Repository/.ruby-version)`
+
 **_Solution:_** Run the command `rbenv install <version number>`
 
+**Error:**
+
+```bash
+== Preparing database ==
+    Sorry, you can't use byebug without Readline. To solve this, you need to
+    rebuild Ruby with Readline support. If using Ubuntu, try `sudo apt-get
+    install libreadline-dev` and then reinstall your Ruby.
+rails aborted!
+LoadError: dlopen(/Users/nickytonline/.rbenv/versions/2.6.1/lib/ruby/2.6.0/x86_64-darwin18/readline.bundle, 9): Library not loaded: /usr/local/opt/readline/lib/libreadline.7.dylib
+```
+
+**_Solution:_** Run `ln -s /usr/local/opt/readline/lib/libreadline.dylib /usr/local/opt/readline/lib/libreadline.7.dylib` from the command line then run `bin/setup` again.
```

Comment:
Keep in mind that the version of readline depends on the system. For example I have readline 8:

```shell
➜  ~ ll /usr/local/opt/readline/lib/libreadline*
Name
/usr/local/opt/readline/lib/libreadline.8.0.dylib
/usr/local/opt/readline/lib/libreadline.8.dylib -> libreadline.8.0.dylib
/usr/local/opt/readline/lib/libreadline.a
/usr/local/opt/readline/lib/libreadline.dylib -> libreadline.8.0.dylib
```